VotingApplication/VotingApplication.Web.Api/Scripts/Manage.js

```diff
@@ -135,9 +195,16 @@
             });
         }
 
+        self.isSignedIn = function () {
+            return signedIn();
```

Comment:
Is there no way to pass this into the constructor? Seems bad to have this floating as a global function.